src/test/shell/bazel/bazel_windows_example_test.sh

```diff
@@ -196,6 +196,30 @@ function test_native_python() {
   assert_test_fails //examples/py_native:fail
 }
 
+function test_native_python_with_runfiles() {
+  BUILD_FLAGS="--experimental_enable_runfiles --build_python_zip=0"
+  bazel build -s --verbose_failures $BUILD_FLAGS //examples/py_native:bin \
+    || fail "Failed to build //examples/py_native:bin with runfiles support"
+  (
+  # Clear runfiles related envs
```

Comment:
To help readability, please indent the lines running in a subshell (line 204 .. 215).

Reply:
Done